components/test_pd/src/server.rs

```diff
@@ -182,6 +183,16 @@ impl<C: PdMocker> Clone for PdMock<C> {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(rename_all = "kebab-case")]
+pub struct ResourceGroupConfig {
+    id: u64,
+    name: String,
+    cpu_quota: f64,
```

Comment:
we also need to supports RU.

Reply:
Okay I will change it to [`GroupSettings`](https://github.com/pingcap/kvproto/blob/master/proto/resource_manager.proto#L109-L112)

Reply:
also necessary to modify the pd struct written to etcd.